code/controllers/subsystem/shuttle.dm

```diff
@@ -170,6 +119,11 @@ SUBSYSTEM_DEF(shuttle)
 		if(MC_TICK_CHECK)
 			break
 
+/datum/controller/subsystem/shuttle/proc/wipe_transit()
```

Comment:
As far as I can tell this proc is unused, unless it's intended to be left in for debugging.

Reply:
Admins always had this as a verb, so I left it in.